jib-core/src/main/java/com/google/cloud/tools/jib/configuration/ContainerConfiguration.java

```diff
@@ -72,7 +72,8 @@ public Builder setProgramArguments(@Nullable List<String> programArguments) {
       if (programArguments == null) {
         this.programArguments = null;
       } else {
-        Preconditions.checkArgument(!programArguments.contains(null));
+        Preconditions.checkArgument(
+            !programArguments.contains(null), "args list contains null elements");
```

Comment:
Since this is in the `ContainerConfiguration` code, should we change these to reference the parameter name explicitly? Like "programArguments contains null elements"

Reply:
Using the name might be weird to plugin users since the naming is slightly different. But maybe we should still be more explicit and say "program arguments" instead of "args"?

Reply:
It's mostly that since this is Jib Core, the language should not be dependent on plugin users. A more robust way to do this would be to have the error message be on the plugin side as part of validation before feeding it into `ContainerConfiguration` here. The precondition check is just a programmer error check rather than a validation check that should be user-facing.